sandstone/filewatcher.py

```diff
@@ -0,0 +1,58 @@
+from watchdog.observers import Observer
+from sandstone.lib.broadcast.manager import BroadcastManager
+from sandstone.lib.broadcast.message import BroadcastMessage
+import watchdog
+
+class FilesystemEventHandler(watchdog.events.FileSystemEventHandler):
+    """
+    Subclass of `watchdog.events.FilesystemEventHandler`
+    Manages on_created, on_deleted, on_moved events
+    """
+    def on_created(self, event):
+        """
+        Event Handler when a file is created
+        """
+        key = 'filetree:created_new_file'
+        data = {
+            'filepath': event.src_path
+        }
+        bmsg = BroadcastMessage(key=key, data=data)
+        BroadcastManager.broadcast(bmsg)
+
+    def on_deleted(self, event):
+        """
+        Event Handler when a file is deleted
+        """
+        key = 'filetree:deleted_file'
+        data = {
+            'filepath': event.src_path
+        }
+        bmsg = BroadcastMessage(key=key, data=data)
+        BroadcastManager.broadcast(bmsg)
+
+class Filewatcher(object):
+    """
+    Starts a watchdog instance to watch over the filesystem
+    Handles events - CREATE, REMOVE,MOVE
+    """
+    _watches = {}
+    _observer = Observer()
+    _event_handler = FilesystemEventHandler()
+    _observer.start()
```

Comment:
The `start()` method doesn't return an instance of `Observer`. I doubt we can chain the methods